curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md

```diff
@@ -18,9 +18,9 @@ That illustrates the difference between an "inline" element and a "block" elemen
 
 By using the inline `span` element, you can put several elements on the same line, and even style different parts of the same line differently.
 
-Nest the word "love" in your "Things cats love" element below within a `span` element. Then give that `span` the class `text-danger` to make the text red.
+Nest the word `love` in your `Things cats love` element below within a `span` element. Then give that `span` the class `text-danger` to make the text red.
```

Comment:
Using a `span` element, nest the word `love` inside the `p` element that currently has the text `Things cats love`. Then give the `span` the class `text-danger` to make the text red.